components/form.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import Link from 'next/link';
+ 
+export default class Form extends React.Component {
+    constructor() {
+        super();
+        this.state = { token: '', formFields: [], formData: {} };
+    }
+ 
+    // Make state targets equal to the value of input fields
+    change = e => {
+        this.setState({
+            formData: {
+                ...this.state.formData,
+                [e.target.name]: e.target.value
+            }
+        });
+    };
+ 
+    // Fetch the data form fields from API, use it in state
+    componentDidMount() {
+        fetch(
+            'https://rominacsvsms.api.ushahidi.io/api/v3/forms/14/attributes?order=asc&orderby=priority'
+        )
+            .then(res => res.json())
+            .then(data => {
+                this.setState({
+                    formFields: data.results
+                });
+            });
+ 
+        if (!this.state.token) {
+            fetch('https://rominacsvsms.api.ushahidi.io/oauth/token', {
+                method: 'POST',
+                body: JSON.stringify({
+                    scope: '*',
+                    client_secret: '35e7f0bca957836d05ca0492211b0ac707671261',
+                    client_id: 'ushahidiui',
+                    grant_type: 'password',
+                    password: 'hellohello',
```

Comment:
add a basic login form to get this password/username instead of hardcoding it here ? 
I invalidated this user in the meantime (in my deployment).